nn_transfer/test/test_architectures.py

```diff
@@ -17,16 +17,19 @@ def setUp(self):
         self.test_data_unet = np.random.rand(1, 1, 224, 224)
 
     def test_simplenet(self):
+        set_seeds()
```

Comment:
You can use the [`setUp` callback](https://docs.python.org/2/library/unittest.html#unittest.TestCase.setUp) so you don't have to repeat it in every test case.